netbout-web/src/test/java/com/netbout/rest/bout/TkInviteTest.java

```diff
@@ -88,4 +91,46 @@ public void invitesUserByEmail() throws Exception {
             throw ex;
         }
     }
+
+    /**
+     * TkInvite should throw an RsFailure when invite alias is too long.
+     * @throws Exception If there is some problem inside
+     */
+    @Test (expected = RsFailure.class)
+    public void inviteWithTooLongAlias() throws Exception {
+        final MkBase base = new MkBase();
+        final String urn = "urn:test:1";
+        final User user = base.user(new URN(urn));
+        final String name = "12345678901234567890123456789012345678901234"
```

Comment:
@jhyle String concat is a bad idea, see [this article](http://www.yegor256.com/2014/06/19/avoid-string-concatenation.html) for alternatives.